gradle/publish-bintray.gradle

```diff
@@ -12,72 +12,149 @@ apply plugin: "com.github.johnrengelman.shadow"
 
 apply from: project.rootProject.file('gradle/maven-central.gradle')
 
-def platform = platformOf(project)
-def coroutines_core = platformLib("kotlinx-coroutines-core", platform)
-
 // ------------- tasks
 
-def isNative() { return project.name.endsWith("native") }
 def bUser = project.hasProperty('bintrayUser') ? project.property('bintrayUser') : System.getenv('BINTRAY_USER')
 def bKey = project.hasProperty('bintrayApiKey') ? project.property('bintrayApiKey') : System.getenv('BINTRAY_API_KEY')
 
+task stubSources(type: Jar) {
+    classifier = 'sources'
+}
+
+task stubJavadoc(type: Jar) {
+    classifier = 'javadoc'
+}
+
+task emptyJar(type: Jar) {
+}
+
 task sourcesJar(type: Jar) {
     classifier = 'sources'
-    if (!isNative()) {
+    if (project.name == "kotlinx-coroutines-core") {
+        from kotlin.sourceSets.commonMain.kotlin
+    } else {
         from sourceSets.main.allSource
     }
+}
 
-    if (project.name == coroutines_core && platform != "common") {
-        // add common source, too
-        from rootProject.file("common/$project.name-common/src/main/kotlin")
+def configureTransitiveDependencies = { Project project, Publication publication ->
```

Comment:
Something is wrong with code. For example, `kotlinx-coroutines-jdk8-1.1.0-SNAPSHOT.pom` file has the following dependencies:
* `kotlin-stdlib` - Ok
* `kotlinx-coroutines-core-native` - Wrong.
* `kotlinx-coroutines-jdk8` - Wrong (but that seems to be a bug in different place).

Reply:
I is a bug in multiplatform + single target plugins, the only solution is to rewrite poms by hand